### ISSUE TITLE
Integration for accessibility on linux

### ISSUE THREAD

**Issue by joshjama** [rhasspy/larynx#28]
On linux blind and visualy impared people use there computer via the screen-reader orca. It reads the contents on the screen out loud ot to the customer. It works with speech-dispatcher. Speech-dispatcher has generic module-files where we can add the integration of larynx relatively easily. To be able to use this natural sounding voices with orca we need to write such a module file. We also need to achief a verry small delay from sending the text to the engine and play the wav, because otherwise the system will not be fluent. But if we achief this we will bring accessibility on linux to the next level.

**Comment by synesthesiam** [comment]
Yes, I've actually already started down this path on a related project :slightly_smiling_face: 

I have a version of Larynx working with Orca, and I've got the delay down very low by keeping a small server running and caching results. I'll post here again when I've uploaded my results.

**Comment by hkalbasi** [comment]
Speech dispatcher is also used by things other than screen readers, for example firefox. So it would be great to support it.

**Comment by jonahkagan** [comment]
Another vote for this. Would love to use Larynx with speech-dispatcher in order to plug it into the WebSpeechAPI and use it for a browser-based screen reader. Would love even some basic pointers on how you set it up @synesthesiam.

**Comment by king-dahmanus** [comment]
not only that, but integrating larynx with speech dispatcher allows orca to
use it as well, great idea!

**Comment by synesthesiam** [comment]
I have a version of Larynx working with speech-dispatcher :)
This and some significant performance/voice quality increases are in the works for Larynx 2.0!

**Comment by king-dahmanus** [comment]
Great news mr michael!
